src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -185,39 +183,9 @@ const formConfig = {
     certifierInformation: {
       title: 'Signer information',
       pages: {
-        page1: {
-          // initialData: mockData.data,
-          path: 'signer-type',
-          title: 'Which of these best describes you?',
-          uiSchema: {
-            ...titleUI('Your information'),
-            certifierRole: radioUI({
-              title: 'Which of these best describes you?',
-              required: () => true,
-              labels: {
-                applicant: 'I’m applying for benefits for myself',
-                sponsor:
-                  'I’m a Veteran applying for benefits for my spouse or dependents',
-                other:
-                  'I’m a representative applying for benefits on behalf of someone else',
-              },
-            }),
-            ...thirdPartyInfoUiSchema,
-          },
-          schema: {
-            type: 'object',
-            required: ['certifierRole'],
-            properties: {
-              titleSchema,
-              certifierRole: radioSchema(['applicant', 'sponsor', 'other']),
-              ...thirdPartyInfoSchema,
-            },
-          },
-        },
         page2: {
           path: 'signer-info',
           title: 'Certification',
-          depends: formData => get('certifierRole', formData) === 'other',
```

Comment:
With the removal of the screener page, we no longer have a `certifierRole` property. Since we want to always show any page that was previously just for 3rd party signers, this and the other `depends` functions that looked at `certifierRole` are dropped.